### PR TITLE
Closes #95

### LINKED ISSUE #95: Reorganize GitHub pages into service and living lab categories
Also add a section for inviting contributions

### PR DESCRIPTION
**Pull request by kbalog**
I have a question concerning the digest email: AFAIU interleaving always happens daily. What about users who chose only weekly digest emails? What content will be shown to them?

### CODE REVIEW

**Comment by oyvi96** [comment]
At the moment they will get the exact same content as those who get daily mail, except that the daily mail contains top 3 each day for the current day. While weekly contains top 3 each day for the entire week by date, which becomes 15 articles like this.

Monday
   - 1
   - 2
   - 3
Tuesday
   - 1
   - 2
   - 3
...

**Comment by kbalog** [comment]
I see. I added a sentence on this.

File: Living_lab.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  * System performance is monitored continuously over time. For system comparison, performance is measured during a designated evaluation period.
+
+
+## Components
+
+The evaluation framework consists of the following components:
+
+  * [arXivDigest API](/arxivdigest/api/)
+    - A RESTful service for experimental recommender systems to access article and user data and to upload personalized recommendations.
+  * [Scraper](scraper/)
```
Comment (oyvi96) [inline Living_lab.md]: Moved to scripts/

File: Living_lab.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    - It continuously monitors the arXiv RSS feed and downloads new articles.
```
Comment (oyvi96) [inline Living_lab.md]: Does not actually monitor continuously, but rather checks on a configured (daily) interval, don't know how specific we need to be though.

File: arxivdigest/api/README.md [hunk 3]
```diff
@@ -1,21 +1,11 @@
-# ArXivDigest API
+# arXivDigest API
```
[the remark this answers is not in the document]
Comment (kbalog) [reply arxivdigest/api/README.md]: Indeed, titles should be capitalized

[the remark this answers is not in the document]
Comment (kbalog) [reply arxivdigest/api/README.md]: Sentence start is the only exception; in all other cases it's written "arXivDigest"

File: Living_lab.md (hunk 1 above)
Comment (kbalog) [reply Living_lab.md]: Updated

File: Living_lab.md (hunk 2 above)
Comment (kbalog) [reply Living_lab.md]: Grouped all batch processes

**Comment by kbalog** [comment]
@oyvi96 Can you please look at this so that I can roll out the changes?

**Pull request merged**